docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -318,6 +318,36 @@ sometimes resolved by additionally installing the ``apptainer-setuid`` package.
       ``slurm.slots_per_node`` to enable multiple CPUs to be used on each node. Without
       ``slurm.slots_per_node`` the job will request ``slots_per_trial`` nodes.
 
+******************
+ PBS Known Issues
+******************
+
+-  The resource pool card on the Cluster tab shows the ``Accelerator`` field as ``unconfigured``.
+   This happens when the GPU type information is not set in the PBS ``resources_available`` object.
+
+   HPC system administrators can set this information using the following set of commands:
```

Comment:
The docs do not use the word 'card' when referring to resources pools displayed on the cluster tab.
That term confused me.

Reply:
Updated the text to be simple and clear.